### PR TITLE
Update 6 years old README on fetch/http-cache

### PR DESCRIPTION
**Pull request by Jxck**
- edge is now based on chromium
- new RFC9111 obsoletes RFC7234
- Fix renamed link

### CODE REVIEW

**Comment by mnot** [comment]
Is it worth mentioning cache-tests.fyi?

**Comment by Jxck** [comment]
@mnot Can I make sure the difference of purpose between wpt and cache-test.fyi ?

**Comment by Jxck** [comment]
@mnot added.

File: fetch/http-cache/README.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
   in Fetch; tests are included here for interest only.
 
 * Some browser caches will behave differently when reloading /
   shift-reloading, despite the `cache mode` staying the same.
 
-* At the moment, Edge doesn't appear to using HTTP caching in conjunction
-  with Fetch at all.
-
+* [cache-tests.fyi](https://cache-tests.fyi/) is yet another test suites of http caching
+  which also includes Server/CDN implementation.
```
Comment (annevk) [inline fetch/http-cache/README.md:26]:
```suggestion
* [cache-tests.fyi](https://cache-tests.fyi/) is another test suite of HTTP caching
  which also caters to server/CDN implementations.
```

**Review by annevk — APPROVED** [review]

**Pull request merged**